app/views/layouts/application.html.erb

```diff
@@ -10,14 +10,15 @@
 
   </head>
 
-  <nav>
-    <ul>
-      <li id='psklrmrfi-link'><%= link_to 'PSKLRMRFI', robots_path %></li>
-      <li><%= link_to 'I Love Mutants!', 'https://en.wikipedia.org/wiki/Amish' %></li>
-      <li><%= link_to 'new order', manufacturers_path %></li>
-      <li><%= link_to 'logout', logout_path, method: :delete %></li>
-    </ul>
-  </nav>
+  <% if user_logged_in? %>
+    <nav>
+      <ul>
+        <li id='psklrmrfi-link'><%= link_to 'PSKLRMRFI', robots_path %></li>
```

Comment:
@justGrasse Why did you remove the Wikipedia link? The commissioner said he liked it.